tests/mol-tts_single/EPUB/package.opf

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Media Overlays</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2022-02-17</dc:date>
+    <dc:description>Tests whether the Reading System implements Text-to-Speech as a fallback when no audio content is provided. Only one fragment is defined in the SMIL file.</dc:description>
+    <dc:identifier id="pub-id">mol-tts_single</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>Single text-to-speech fallback for MO</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="belongs-to-collection">should</meta>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#sec-text-to-speech</meta>
```

Comment:
Same comment — I think this should be one or two more specific anchors, since there are two normative statements here.

Reply:
+1, like above